README.md

```diff
@@ -119,12 +119,12 @@ if !shouldDoTheThing {
 
 ---
 
-#####Use `where` to eliminate layers of nesting.
+#####Use a comma to eliminate layers of nesting.
 
 *Preferred:*
 
 ```swift
-if let someInt = optionalInt where someInt % 2 == 0 {
+if let someInt = optionalInt , someInt % 2 == 0 {
```

Comment:
The space in front of this comma should be removed. The migrator literally just replace "where" with ",", but I find it awkward with the space in the front.